cli/src/test/java/com/devonfw/tools/ide/commandlet/InstallCommandletTest.java

```diff
@@ -68,7 +68,8 @@ public void testInstallCommandletRunWithVersionAndVersionIdentifier(WireMockRunt
     install.run();
     // assert
     assertTestInstall(context);
-    assertThat(context).logAtSuccess().hasMessage("Successfully installed java in version " + version21 + " replacing previous version " + version17);
+    assertThat(context).logAtSuccess()
+        .hasMessage("Successfully installed java in version " + version21 + " replacing previous version " + version17 + " of java");
```

Comment:
` of java` is now added to the log message.
May seem redundant in this example, but not if we have:
```
Successfully installed intellij in version 2025.2.5 replacing previous version 2025.2.5 of intellij/ultimate
```